recipes-bsp/firmware-imx/firmware-ele-imx_0.1.3.bb

```diff
@@ -37,7 +36,4 @@ ALLOW_EMPTY:${PN}-ext = "1"
 FILES:${PN} += "${nonarch_base_libdir}/firmware/imx/ele/${SECO_FIRMWARE_NAME}"
 FILES:${PN}-ext += "${nonarch_base_libdir}/firmware/imx/ele/${SECOEXT_FIRMWARE_NAME}"
 
-RREPLACES:${PN} = "firmware-sentinel"
-RPROVIDES:${PN} = "firmware-sentinel"
-
```

Comment:
Perhaps we don't want to do this. It looks like an improvement made in the meta-freescale that was not pushed down to meta-imx

Reply:
Do you prefere to keep it in meta-freescale?

Reply:
I do, just wanted to get @otavio's agreement as well.

Reply:
Sure, we should keep those. Those are to support the upgrade using package feeds and we should not drop those.

Reply:
It kept, done.